src/videojs-hls.js

```diff
@@ -331,7 +331,7 @@ videojs.Hls.prototype.addCuesForMetadata_ = function(segmentInfo) {
       frame = metadata.frames[i];
       time = segmentOffset + ((metadata.pts - minPts) * 0.001);
       cue = new window.VTTCue(time, time, frame.value || frame.url || '');
```

Comment:
WebKitDataCue is not available in Chrome/FireFox. We have to stick with VTTCue for now. There is an issue filed for this: https://github.com/videojs/videojs-contrib-hls/issues/303